.eslintrc

```diff
@@ -1,4 +1,23 @@
 {
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "prettier",
+    "prettier/react"
+  ],
+  "parser": "babel-eslint",
+  "plugins": [
+    "react",
+    "prettier",
+    "babel",
+    "no-only-tests"
+  ],
+  "ignorePatterns": [
+    "node_modules",
```

Comment:
I guess you now removed the .eslintignore file, which is fine if you prefer to only use the .eslintrc

But adding node_modules in ignorePatterns is not needed since it is already automatically ignored by eslint, see https://eslint.org/docs/user-guide/configuring#eslintignore

Reply:
I like explicitly having it here since we have other ignore patterns.
Just a piece of mind, and I don't have to remember if it does it by default or not.